src/main/webapp/trip/tripDetails.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>New Trip</title>
+         
+        <!-- Stylesheet --> 
+        <link rel="stylesheet" href="../style.css">
+
+        <!-- Fonts -->
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+
+        <!-- Favicon -->
+        <link rel="icon" href="images/favicon.ico">
+    </head>
+
+    <body>
+        <div class="container">
+            <h1 class="home-title">Trip name</h1>
+            <h3 class="home-subtitle">Add participants</h3>
+            <form action="NewPeople" method="POST">
+                <div class="inputBox">
+                    <input name="text-input-participant" type="text" placeholder="" required="required">
```

Comment:
text-input-participants (plural, since it can be multiple)

Reply:
Also if this is a list of name/ids? ca n you specify that in the placeholder text